### PR TITLE
Add missing bounds as present on CHaP

### PR DESCRIPTION
**Pull request by andreabedini**
These are working bounds based on the current build plan. They have been [applied to the released version](https://github.com/input-output-hk/cardano-haskell-packages/pull/154) of these packages on chap.

### CODE REVIEW

File: freer-extras/freer-extras.cabal [hunk 1]
```diff
@@ -109,4 +109,4 @@ test-suite freer-extras-test
     , semigroups
     , sqlite-simple
     , tasty
-    , tasty-hedgehog
+    , tasty-hedgehog1
```
Comment (koslambrou) [inline freer-extras/freer-extras.cabal:112]: Typo?

Comment (andreabedini) [reply freer-extras/freer-extras.cabal:112]: fuuuuuuu, this ended up in CHaP as well :facepalm: yes, it's a typo

**Comment by andreabedini** [comment]
Alright, I defintely have some bounds issues. I was sure I had checked the revisions against the plutus-apps repo but :shrug:. It is all a manual process so it happens that I get things wrong.

**Comment by andreabedini** [comment]
I changed some bounds to make everything work. I will do another revision on CHaP if/when these bounds are accepted.

**Comment by andreabedini** [comment]
Reminder: make a new revision on CHaP if/when these are merged.

**Pull request closed without merging**